src/PaneSystem.tsx

```diff
@@ -46,6 +49,190 @@ const PaneSystem = ({
     setContainerSize({ width, height });
   });
 
+  useNestedPaneSystemChecker();
```

Comment:
Does this work on all cases? It seems working, but I'm not sure about the coverage.

Reply:
Would it be better to write test code?

Reply:
`InnerPaneSystem` explicitly states that it's the sole direct child of `Pane`.
In case of `PaneSystem`, Can we just rely on context drilling to check if PaneSystem is nested in all cases?

Reply:
However, since we're delegating to the presence of `InnerPaneSystem`, there's a possibility that we might not cover all cases as we work on it in the future. (even if we assume that it covers all cases at the moment)

Reply:
We need test code some day, but let's leave it for now :)